### DISCUSSION TITLE
confusing about generating the keystore and cert

### DISCUSSION THREAD

**Discussion by haiyang1991Liu** [eclipse-edc/Connector#2300] [Q&A] [answered]
First question.  where can I generate the keystore and cert and what does it used for?
Second question, It seems useless if I only start the connector  without an DAPS right?
Does there any docs can let me follow step one by one then let me know how these component work together?

**Comment by paullatzelsperger** [comment, marked as answer]
Hi @haiyang1991Liu,
The keystore is used for the `FsVault`, which is a file-system based implementation of a secret store (=vault). Please only use it for testing/development, production use is **not** advised.

DAPS is not a component of EDC, it is merely one of many identity providers that can be used with EDC. Others would be WebDID and OAuth2. Thus, you could use it's respective [documentation](https://github.com/International-Data-Spaces-Association/omejdn-daps) as a starting point.

Please have a look at the `README.md` and the therein linked `launcher/ids-connector/README.md`, which contain instructions on how to setup a DAPS instance. Furthermore, have a look at `DapsIntegrationTest.java` that outlines how the verification process with DAPS works.

From your questions I can see that you clearly have neither read the documentation, nor familiarised yourself with EDC's architecture or the code base. All of these are essential preparatory steps (not just for EDC, for any software, really), the importance of which simply cannot be overstated.

I will lock/close this discussion, as it is non productive and doesn't add any value.
